.devops/repo/src/commands/build.rs

```diff
@@ -0,0 +1,21 @@
+use crate::utils::{execute, print_applications};
+use std::process::Command;
+
+pub(crate) fn run(apps: Vec<String>, root: String) {
+    let mut errors: Vec<String> = vec![];
+
+    for app in apps {
+        println!("\nBuilding {}", app);
+
+        // TODO: safety
+        let project = std::fs::canonicalize(format!("{}/{}/project", root, app)).unwrap();
```

Comment:
You may use `anyhow` here to quickly throw an error:

```suggestion
        let project = std::fs::canonicalize(format!("{}/{}/project", root, app)).map_err(|e| anyhow!("Failed to canonicalize path to project for app '{}': {}", app, e));
```